health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/SearchChecklist.js

```
@@ -240,14 +236,30 @@ const SearchChecklist = () => {
+            <Button
+              label={t("GO_BACK")}
+              title={t("GO_BACK")}
+              variation="secondary"
+              style={{
+                marginLeft: "2.5rem",
+              }}
+              onClick={() => {
+                history.push(`/${window.contextPath}/employee/campaign/view-details?campaignNumber=${campaignNumber}`);
+              }}
+              icon={"ArrowBack"}
+            />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)